modules/ROOT/pages/on-error-scope-concept.adoc

```diff
@@ -3,51 +3,138 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-On Error components define the types of errors that they handle. You configure
-On Error components inside the scope-like Error Handler component.
-
-When an error occurs, the Error Handler component routes the Mule error to the
-first On Error component that matches the error. At this point, the error
-becomes available for inspection, so the On Error component can execute and
-act accordingly.
-
-Mule supports two types of On Error components. Both can perform matching
-through an error type (or group of error types) or through an expression
-(for advanced use cases). Each On Error component specifies a different
-behavior to perform when an error occurs. This behavior affects the
-outcome of its owner (the Flow or Try scope where the Error Handler is defined):
-
-[%header%autowidth.spread,cols='a,a']
-|===
-|On Error Scope |Description
-| On Error Propagate
-| Executes but propagates the error to the next level up, breaking the owner's execution. +
-If a transaction is being handled, it’s rolled back.
-
+When an error occurs in a Mule app, an Error Handler component routes the error
+to the first On-Error component (On Error Continue or On Error Propagate)
+configuration that matches the Mule error, such as HTTP:NOT_FOUND,
+DB:CONNECTIVITY, or ANY (the default for all Mule errors). If no error handling
+is configured for the error, the app follows a
+xref:error-handling.adoc#default_error_handling[default error handling] process.
+
+.Error Handling Components in Studio
+image::error-handlers.png[width=75%]
+
+On-Error components differ in the way they affect their _owner_,
+that is, the Flow or Try scope where they are defined.
+
+On Error Propagate::
+Executes but propagates the error to a higher level, such as a containing
+scope (for example, to a Flow that contains a Try scope where the error occurs)
+or external flow containing a Flow Reference to the flow in which an error occurs.
+The error breaks the owner's execution and propagates to that higher level,
+causing a failure at that level. In addition, any transaction the owner handles
+is rolled back.
++
 image::error-handling-e77ec.png[]
 
-| On Error Continue
-| Executes and uses the result of the execution, as the result of its owner, as if the owner had actually completed the execution successfully. +
-If a transaction is being handled, it would be committed as well.
-|===
-
-Each On Error Scope can be set up to accept a different set of error types. Both in Studio and Design Center, you can pick from a list of errors that could be raised by the components used in the error handler's owner.
+On Error Continue::
+Executes and uses the result of the execution as the result of its owner,
+as if the owner completed the execution successfully. Any transaction the
+owner handles is committed.
```

Comment:
Same: the tx is not commited if the on-error-continue's owner is not the one that created the tx

Reply:
done